build/parents/build/pom.xml

```diff
@@ -0,0 +1,1522 @@
+                                                </goals>
+                                            </pluginExecutionFilter>
+                                            <action>
+                                                <ignore />
+                                            </action>
+                                        </pluginExecution>
+                                        <pluginExecution>
+                                            <pluginExecutionFilter>
+                                                <groupId>de.thetaphi</groupId>
+                                                <artifactId>forbiddenapis</artifactId>
+                                                <versionRange>[3.1,)</versionRange>
+                                                <goals>
+                                                    <goal>check</goal>
+                                                    <goal>testCheck</goal>
+                                                </goals>
+                                            </pluginExecutionFilter>
+                                            <action>
+                                                <ignore />
+                                            </action>
+                                        </pluginExecution>
+                                        <pluginExecution>
+                                            <pluginExecutionFilter>
+                                                <groupId>
+                                                    org.asciidoctor
+                                                </groupId>
+                                                <artifactId>
+                                                    asciidoctor-maven-plugin
+                                                </artifactId>
+                                                <versionRange>
+                                                    [2.2.1,)
+                                                </versionRange>
+                                                <goals>
+                                                    <goal>
+                                                        process-asciidoc
+                                                    </goal>
+                                                </goals>
+                                            </pluginExecutionFilter>
+                                            <action>
+                                                <ignore></ignore>
+                                            </action>
+                                        </pluginExecution>
+                                    </pluginExecutions>
+                                </lifecycleMappingMetadata>
+                            </configuration>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
+    </profiles>
```

Comment:
These probably belong in the root POM? Only the last parts of the IDEA profile (dependencies and build) really need to be here, and you can just split the profile.

Reply:
Yeah, I wanted to keep these IDE profiles together in one place, but this should work too 😃